.github/CODEOWNERS

```diff
@@ -10,6 +10,9 @@ src/applications/proxy-rewrite/redirects/crossDomainRedirects.json @department-o
 src/applications/find-va-forms @department-of-veterans-affairs/vsa-frontend
 src/platform/site-wide @department-of-veterans-affairs/vsa-frontend
 src/applications/yellow-ribbon @department-of-veterans-affairs/vsa-frontend
+src/applications/personalization @department-of-veterans-affairs/vsa-frontend
+src/applications/disability-benefits @department-of-veterans-affairs/vsa-frontend
+src/platform/user/profile/vet360 @department-of-veterans-affairs/vsa-frontend
```

Comment:
I kinda want to add `src/applications/static-pages` for Public Websites, but sometimes code there can reach a scope that seems like should be reviewed by VSP.

Reply:
when do you think it should be reviewed by VSP?

Reply:
Good question, I'll have to look into it and report back. In general, the static-pages bundle is sort of a centralized place for changes outside of a certain app, so that's why I think VSP approval should happen. But then again, most of the changes aren't platform-ish, so maybe not